homeassistant/components/enphase_envoy/__init__.py

```diff
@@ -64,4 +64,11 @@ async def async_remove_config_entry_device(
         for inverter in envoy_data.inverters:
             if str(inverter) in dev_ids:
                 return False
+    if envoy_data and envoy_data.encharge_inventory:
```

Comment:
Should we make 1 if statement for `if envoy_data` and then have the other 3 inside?

Reply:
Done